pcsx2/Pcsx2Config.cpp

```diff
@@ -1896,6 +1897,14 @@ bool EmuFolders::ShouldUsePortableMode()
 	return FileSystem::FileExists(Path::Combine(AppRoot, "portable.ini").c_str()) || FileSystem::FileExists(Path::Combine(AppRoot, "portable.txt").c_str());
 }
 
+std::string EmuFolders::GetPortableModePath()
+{
+	const auto portable_txt_path = Path::Combine(AppRoot, "portable.txt");
+	const auto portable_path = FileSystem::ReadFileToString(portable_txt_path.c_str());
+	const auto trimmed_path = StringUtil::StripWhitespace(portable_path.value_or(""));
```

Comment:
I was having trouble getting this to work. Turns our the issue was the fact that `value_or("")` returns a newly constructed `std::string` and `StripWhitespace` takes in and returns a `string_view`. `string_view` is essentially a pointer with no ownership so the `value_or` string was destroyed, making the returned `string_view` point to garbage.

Can you try this instead?
```cpp
const auto portable_path = FileSystem::ReadFileToString(portable_txt_path.c_str()).value_or("");
const auto trimmed_path = StringUtil::StripWhitespace(portable_path);
```

Reply:
Done, thanks. I was making a few last minute adjustments to make sure extra newlines or other whitespace from the file would be excluded and missed that.